mit_tab_bot/api.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from simple_rest_client.api import API
+
+def construct_api():
+    api = API(
+        api_root_url='http://%(tournament_name)s.nu-tab.com/discord/' % {
+            'tournament_name': os.environ.get('TOURNAMENT_NAME', 'NONAME')
+        },
+        json_encode_body=True
+    )
+
+    api.add_resource(resource_name='debaters')
+    api.add_resource(resource_name='judges')
+    api.add_resource(resource_name='rooms')
+    api.add_resource(resource_name='rounds')
+
+    return api
+
+
+def get_judge(discord_id):
+    api = construct_api()
+
+    response = api.judges.list(
+        params={'discord_id': discord_id}
+    )
+    return response.body[0] if len(response.body) > 0 else None
```

Comment:
nit: can just be `if response.body` or `if len(response.body)`

Reply:
lol yes